src/components/application_manager/src/commands/hmi/on_seek_media_clock_timer_notification.cc

```diff
@@ -53,7 +53,15 @@ void OnSeekMediaClockTimerNotification::Run() {
   (*message_)[strings::params][strings::function_id] =
       static_cast<int32_t>(mobile_apis::FunctionID::OnSeekMediaClockTimerID);
 
-  SendNotificationToMobile(message_);
+  const ApplicationSet& accessor =
+      application_manager_.applications().GetData();
+
+  ApplicationSetIt it = accessor.begin();
+  for (; accessor.end() != it;) {
+    const ApplicationSharedPtr app = *it++;
```

Comment:
@indlu, this expression does not look so obvious.
Please, move increment operation to head of `for` loop.